{{cookiecutter.repostory_name}}/bin/backup-db.sh

```diff
@@ -19,7 +19,7 @@ else
   $DUMP_DB_TO_STDOUT > "$TARGET"
 
   if [ -n "${EMAIL_HOST:-}" ] && [ -n "${EMAIL_TARGET:-}" ]; then
-    ${SCRIPT_DIR}/backup-db-to-email.sh "${TARGET}"
+    "${SCRIPT_DIR}"/backup-db-to-email.sh "${TARGET}"
```

Comment:
is this what the shellcheck recommended? I have never seen quotes like this, usually it's the whole path in quotes instead of just the variable. Did it give any explanation why is it recommended this way? 🤔

Reply:
non explanation was given, they are AFAIK practicall identical and typically, I also prefer `"${SCRIPT_DIR}/backup-db-to-email.sh"` form, but what I prefer even more is not carring about this detail :D